infra/blueprint-test/pkg/cai/cai.go

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Package cai provides a set of helpers to interact with Cloud Asset Inventory
+package cai
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/gcloud"
+	"github.com/tidwall/gjson"
+)
+
+type CmdCfg struct {
+	sleep     int      // minutes to sleep prior to CAI retreval. default: 2
+	assetType string   // asset type to retrieve. default: all
```

Comment:
Not super familiar with CAI but would this allow to retrieve two asset types or should it be slice of strings?

Reply:
Good idea - I'll also need to re-work the newCmdConfig, but the underlying ` --asset-types` can accept a comma delimitated list.